### PR TITLE
Reduce risk of CVD based on various interventions

### PR DESCRIPTION
**Pull request by Abdulrahims**
Reduce the cardiovascular disease, atrial fibrillation, and stroke risks based on a CHW intervention providing a given service. CVD related services in this PR include:

* Blood pressure screenings
* Healthy diet and physical activity counseling
* Obesity screenings
* Added CVD measures to tobacco and alcohol misuse screenings
* Aspirin medication
* Statin medication

### CODE REVIEW

File: src/main/java/org/mitre/synthea/modules/CommunityHealthWorker.java [hunk 1]
```diff
@@ -241,5 +251,117 @@ private void lungCancerScreening(Person person, long time)
...
+
+			Procedure ct = person.record.procedure(time, "Blood pressure screening - first call (procedure)");
+			
+			ct.codes.add(new Code("SNOMED-CT","185665008","Blood pressure screening - first call (procedure)"));
+			
+			double blood_pressure_chw_delta = Double.parseDouble( Config.get("lifecycle.blood_pressure.chw_delta", "0.1"));
+
+			if(person.attributes.containsKey("cardio_risk")){
+				double cardioRisk = (double) person.attributes.get("cardio_risk");
+				cardioRisk = cardioRisk / (2 + blood_pressure_chw_delta);
```
Comment (dehall) [inline src/main/java/org/mitre/synthea/modules/CommunityHealthWorker.java]: dividing the risk by 2+ seems high. do we have any source for these impacts?

File: src/main/java/org/mitre/synthea/modules/CommunityHealthWorker.java [hunk 2]
```diff
@@ -193,6 +195,10 @@ private void tobaccoScreening(Person person, long time)
 	{
 		if((boolean) person.attributes.getOrDefault(Person.SMOKER, false) && this.offers(TOBACCO_SCREENING)) 
 		{
+			Procedure ct = person.record.procedure(time, "Tobacco usage screening (procedure)");
```
Comment (dehall) [inline src/main/java/org/mitre/synthea/modules/CommunityHealthWorker.java]: I think the "if" check here needs to be more robust and this is as good a time as any to make a change even if it wasn't the original intent of this PR. Tobacco usage screening shouldn't be based on this flag where we know they are a smoker. 

From USPSTF (emphasis mine): "The USPSTF recommends that clinicians ask all adults about tobacco use, advise them to stop using tobacco, and provide behavioral interventions and U.S. Food and Drug Administration (FDA)–approved pharmacotherapy for cessation to adults who use tobacco."

File: src/main/java/org/mitre/synthea/modules/CommunityHealthWorker.java [hunk 3]
```diff
@@ -206,6 +212,10 @@ private void alcoholScreening(Person person, long time)
 	{
 		if((boolean) person.attributes.getOrDefault(Person.ALCOHOLIC, false) && this.offers(ALCOHOL_SCREENING))
```
Comment (dehall) [inline src/main/java/org/mitre/synthea/modules/CommunityHealthWorker.java]: Similar comment here - screening should be based on some criteria, not only if they actually are alcoholic.

"The USPSTF recommends that clinicians screen adults age 18 years or older for alcohol misuse and provide persons engaged in risky or hazardous drinking with brief behavioral counseling interventions to reduce alcohol misuse."

**Comment by dehall** [comment]
My only concern here is the level of impact that CHWs have. The numbers seem large so do we have any sources for these? If so could you list those as code comments, or if not could you add a comment or TODO to clarify?

As a general comment, I think we have to be careful not to have the CHWs do too much - their capabilities are limited and not magic cure-alls. A CHW taking your blood pressure isn't going to automatically reduce your risk of heart attack. If they find you are at risk, they can offer suggestions (diet and exercise) to slowly get you back on track, or refer you to a licensed provider to get a medication, which will reduce blood pressure directly.

**Comment by jawalonoski** [comment]
I still need to review in detail, but the numbers can get tweaked. Having big effects to start with can be useful to demonstrate that the system is blatantly altered by the CHW. After that, we can dial back the effectiveness based on better data.

**Pull request merged**